packages/@sanity/base/src/presence/overlay/StickyOverlay.tsx

```diff
@@ -173,11 +172,23 @@ export function StickyOverlay(props: Props) {
       return (
         <>
           {[
-            renderDock('top', margins, grouped.top, counts.nearTop),
+            <PresenceDock
```

Comment:
Perhaps we could avoid the array around here now?

Reply:
Sure!